templates/index.html

```diff
@@ -15,6 +15,8 @@
 {{ sources['flickr'].button_html('listen')|safe }}
 {{ sources['github'].button_html('listen')|safe }}
 {{ sources['mastodon'].button_html('listen')|safe }}
```

Comment:
How can I add one of these for `/meetup/publish/start`?

Reply:
just blindly copying and modifying one of these lines should work, right?